publication-event-handlers/src/main/java/no/unit/nva/publication/events/handlers/expandresources/PersistedDocumentMetadata.java

```diff
@@ -0,0 +1,73 @@
+package no.unit.nva.publication.events.handlers.expandresources;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Objects;
+import no.unit.nva.expansion.model.ExpandedDatabaseEntry;
+import no.unit.nva.expansion.model.ExpandedDoiRequest;
+import no.unit.nva.expansion.model.ExpandedMessage;
+import no.unit.nva.expansion.model.ExpandedResource;
+import no.unit.nva.identifiers.SortableIdentifier;
+import nva.commons.core.JacocoGenerated;
+
+public class PersistedDocumentMetadata {
```

Comment:
Collect all information about a persisted document in one object. This will allow us to deprecate fields and use create new fields.